src/components/Header.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import styled from "styled-components";
+
+import SearchOption from "./SearchOption";
+
+const StyledHeader = styled.div`
+	padding-inline: 52px;
+	padding-block: 20px;
+	align-items: center;
+	display: flex;
+	justify-content: space-between;
+`;
+
+const StyledTitle = styled.h2`
+	text-align: center;
+	font-size: 18px;
+`;
+
+const Header = (props) => {
```

Comment:
Great if you consider the option to destructure the props in function signature. Advantage: DRY code + have a single place where you can see all the props of a component.